README.md

```diff
@@ -1,6 +1,9 @@
 # sfdc-ant-build-script
 Build script for Salesforce projects
 
+### Version Notes
+1. Ant version greater than 1.6, tested with 1.10.1 (latest version).
```

Comment:
Let's kill the new Version Notes section and just move your bullet regarding ant version (line 5) as the first item under the Prerequisites section.